src/main/java/model/response/ServiceIdResponse.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2016, RadiantBlue Technologies, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package model.response;
+
+/**
+ * A Response containing an ID regarding a Piazza Service.
+ * 
+ * @author Patrick.Doody
+ *
+ */
+public class ServiceIdResponse extends PiazzaResponse {
+
+	private String type = "service-id";
```

Comment:
Response for a registration of a service is in the following format:
{
  "type": "service",
  "serviceId": "304e4ef3-f6a0-4f91-98c3-447a4f62d0af"
}

Will this change impact this format?